src/commands/CreateCommand.js

```diff
@@ -0,0 +1,33 @@
+import Command from "../Command";
+import ConfigUtilities from "../ConfigUtilities";
+import FileSystemUtilities from "../FileSystemUtilities";
+import NpmUtilities from "../NpmUtilities";
+import path from "path";
+
+export default class LsCommand extends Command {
```

Comment:
I actually do the same thing: https://github.com/doug-wade/clefs/tree/master/packages/generator-clefs-plugin -- I suppose we could consider having `init` run [generator-gulp-plugin](https://github.com/hparra/generator-gulp-plugin) or something.  However, asking new users to learn about and maintain a yeoman generator seems like a pretty high barrier to entry; I'm inclined to support a simple command in lerna and add the yeoman generator suggestion to the docs.